builder/builder-config/src/test/java/io/helidon/builder/config/StringValueParserHolderTest.java

```diff
@@ -14,22 +14,22 @@
  * limitations under the License.
  */
 
-package io.helidon.pico.builder.config.spi.test;
+package io.helidon.builder.config;
 
-import io.helidon.pico.builder.config.spi.StringValueParserHolder;
+import io.helidon.builder.config.spi.StringValueParserHolder;
+import io.helidon.common.testing.junit5.OptionalMatcher;
 
+import org.hamcrest.CoreMatchers;
 import org.junit.jupiter.api.Test;
 
-import static io.helidon.common.testing.junit5.OptionalMatcher.optionalEmpty;
-import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 class StringValueParserHolderTest {
 
     @Test
     void testGlobalInstanceIsEmpty() {
         assertThat(StringValueParserHolder.stringValueParser(),
-                   is(optionalEmpty()));
+                                 CoreMatchers.is(OptionalMatcher.optionalEmpty()));
```

Comment:
why would you remove static imports and replace them with explicit invocations? Please revert ALL of these

Reply:
pushed